rollup.config.js

```diff
@@ -6,19 +6,17 @@ const { version } = require('./package.json');
 const terserConfig = {
 	compress: true,
 	mangle: true
-}
+};
 
 function bundle(type) {
 	let minify = false;
 	let format = type.replace(".min", () => {
 		minify = true;
-		return ""
+		return "";
 	});
 
-	let suffix = `.${type}`.replace(".esm", "");
-
 	return {
-		file: `dist/jsep${suffix}.js`,
+		file: `dist/${format}/jsep.${type}.js`,
```

Comment:
Please note that `suffix` and `type` are not the same thing. I think there should be a "canonical" `jsep.js` with no format specifier, and I think that should be the ESM version. 
I also think it's a little strange if `/dist/jsep.js` points to nothing because everything is in subfolders. What do you think about `/dist/ containing the ESM build and then all other formats (CJS, IIFE etc) to be in their own subfolders? That would still maintain the same folder structure you have here for them, while also providing nicer URLs for the ESM bundle(s).

Reply:
Fair point. I was trying to get both CJS and ESM working properly without having to specify the full path and it took a few tries to get it working. Since it only requires the one overriding package.json I've put it back to the original dist file structure